src/Microsoft.AspNet.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -109,19 +109,12 @@ public void Add([NotNull] Operation operation, [NotNull] object objectToApplyTo)
         /// Add is used by various operations (eg: add, copy, ...), yet through different operations;
         /// This method allows code reuse yet reporting the correct operation on error
         /// </summary>
-        private void Add(
-            [NotNull] string path,
-            object value,
-            [NotNull] object objectToApplyTo,
-            [NotNull] Operation operationToReport)
+        private void Add([NotNull] string path, object value, [NotNull] object objectToApplyTo, [NotNull] Operation operationToReport)
```

Comment:
formatting change makes this harder to review :frowning: - we use a 120 char line length because that's what works in github's UI.